lighthouse-core/audits/seo/plugins.js

```diff
@@ -41,7 +41,7 @@ const UIStrings = {
   /** Description of a Lighthouse audit that tells the user *why* they need to avoid using browser plugins in their content. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'Search engines can\'t index plugin content, and ' +
     'many devices restrict plugins or don\'t support them. ' +
-    '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/plugins).',
+    '[Learn more](https://web.dev/plugins).',
```

Comment:
FYI the screenshot in this doc is of a Lighthouse error/bug instead of the audit failing